compiler/compiler.go

```diff
@@ -421,6 +436,10 @@ func (c *compileCtx) getBoxedType(t protobuf.Type) protobuf.Type {
 	}
 }
 
+func (c *compileCtx) getArrayType() protobuf.Type {
```

Comment:
This is just returning a constant so it's fine to inline above and just removing this method.